From 8338deeb8fc28925257a9a3bfad1dc78c8d74cef Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.cirrus.com>
Date: Fri, 31 Jul 2026 16:18:26 +0100
Subject: [PATCH 09556/15333] ASoC: es9356: Remove unused headers

es9356 doesn't use any SDCA function/regmap features, remove
the redundant included headers.

Signed-off-by: Charles Keepax <ckeepax@opensource.cirrus.com>
Link: https://patch.msgid.link/20260731151826.961912-1-ckeepax@opensource.cirrus.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/codecs/es9356.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/codecs/es9356.c b/sound/soc/codecs/es9356.c
index a9863900bf70..80db0f2f4aef 100644
--- a/sound/soc/codecs/es9356.c
+++ b/sound/soc/codecs/es9356.c
@@ -20,8 +20,6 @@
 #include <sound/soc.h>
 #include <sound/soc-dapm.h>
 #include <sound/tlv.h>
-#include <sound/sdca_function.h>
-#include <sound/sdca_regmap.h>
 #include <linux/interrupt.h>
 #include <linux/irq.h>
 #include <sound/jack.h>
-- 
2.55.0

